Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py

```diff
@@ -966,6 +966,7 @@ def test_function(client):
     response = client.get_findings()
     if response['ResponseMetadata']['HTTPStatusCode'] == 200:
         return 'ok', {}, {}
+    return None
```

Comment:
Why not return the error code/more informative error?

Reply:
When we will get an error the error will be catch in the main with the `except Exception as e` line.

Reply:
So, in this case, we can remove the None, what happens when we get a response with 400 status code? Is an exception raised?